yt/frontends/chimera/api.py

```diff
@@ -0,0 +1,11 @@
+"""
+API for yt.frontends.chimera
+
+
+
+"""
+
+from . import tests
```

Comment:
this module is meant to represent the parts of the frontend we want to expose to the user. We can't expose test modules because they are not distributed when yt is released
```suggestion
```